.editorconfig

```diff
@@ -1,2 +1,2 @@
-[*.{gradle,mapping}]
+[*.{gradle,mapping,unpick}]
```

Comment:
Does this do something with enigma?

Reply:
It fixes IDEA wanting to use spaces for unpick files.

Reply:
got it